### DISCUSSION TITLE
Scala Setup Troubles

### DISCUSSION THREAD

**Discussion by Seg-Faulted** [scala/scala-swing#175] [Q&A]
Upon `scalac enterfilenamehere.scalac` error provides `value swing is not a member of scala - did you mean scala.Long?`, and this long log: https://cl1p.net/swingLogError, tried adding `libraryDependencies += "org.scala-lang.modules" %% "scala-swing" % "3.0.0"` to build.sbt but does not work, provides value swing is not... 
 I tried manually adding scala-swing in library search in scastie, doesn't work, provides value swing is not...

Any help is appreciated!

**Comment by SethTisue** [comment]
as your troubles (as the title you chose indicates) aren't really specific to this library, you'd probably have better luck at https://users.scala-lang.org (or the Scala Discord, if you prefer chat to forums)

if you do ask elsewhere, you'll want to show your entire `build.sbt`
